### PR TITLE
Add tests for capture_frame_block and find_face_block

### PR DESCRIPTION
**Pull request by cowleyk**

### CODE REVIEW

**Comment by cowleyk** [comment]
For the `capture_frame_block` and `find_face_block`, should I also write a test from when `self.ipcam() = true`?

File: README.md [hunk 1]
```diff
@@ -1,80 +1,69 @@
...
-Grab a frame of video from a specified camera and send the frame data as a signal.
+Commands
+--------
 
 Dependencies
-----------------
-- opencv-python
-- pickle
+------------
 - base64
```
Comment (dorzel) [inline README.md]: There are a few dependencies here that are builtins, `base64`, `pickle`, and `urllib.requests` (I think that was meant to be `urllib.request` anyways). Also get rid of these in requirements.txt

File: get_encoding_from_file_block.py [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 from nio.block.base import Block
 from nio.properties import VersionProperty, StringProperty, ListProperty
 from nio.signal.base import Signal
 from nio.types.string import StringType
 
-import face_recognition
-import pickle
-import base64
 
 class GetEncodingFromFile(Block):
```
Comment (dorzel) [inline get_encoding_from_file_block.py:11]: Unneeded `__init__` here

File: get_encoding_from_file_block.py [hunk 3]
```diff
@@ -23,7 +24,8 @@ def save_encoding(self, file_path, save_name, user_id, file2_path=None):
         for f in file_path:
```
Comment (dorzel) [inline get_encoding_from_file_block.py:21]: Looks like `file2_path` is unused in this function

File: tests/test_capture_frame_block.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            self.configure_block(blk, {})
+            blk.start()
+            blk.process_signals([Signal()])
+            self.assert_num_signals_notified(1)
+            self.assert_last_signal_notified(Signal({
+                'capture': 'signal'
+            }))
+            blk.stop()
+
+        # Does this also need to test when self.ipcam() = True?
```
Comment (dorzel) [inline tests/test_capture_frame_block.py]: I think it would be good to test this, maybe even just asserting that `urllib.request.urlopen()` was called

**Review by dorzel — CHANGES_REQUESTED** [review]

**Pull request merged**